app/javascript/components/molecules/Dropdown.js

```diff
@@ -4,32 +4,16 @@ import DropdownMenuMultiSelect from '../molecules/DropdownMenuMultiSelect'
 import { find } from 'lodash'
 import PropTypes from 'prop-types'
 
-const computeTopWith = (buttonRef) => {
-  // Hardcoded for now.
-  // Not spending time yet on getting the right top based on button
-  return 40
-}
-
-const computeLeftWith = (ref) => {
-  return 0
-}
-
 class Dropdown extends React.Component {
   constructor (props) {
     super(props)
-    this.state = { expanded: false, style: { top: 0, left: 0 } }
+    this.state = { expanded: false, style: { top: 40, left: 0 } }
```

Comment:
Setting the values here, since the previous code was not being used and we use css clases for manipulating the top.